### PR TITLE
Hack: experimentation to see if we can use the bundle locally with Bazel

### PR DESCRIPTION
**Pull request by jhchabran**
This is some discovery around how to integrate the bundle with the frontend service locally, build on top of the other branch to use bazel in sg start. 

None of this is meant to be merged in its present form. 

- The images are not available to the webpack bundle, that's probably something to fix 
- We completely drop the need to go through the webpack server in Caddy 
- We had to tweak a bit the ui/*.go files to find where the assets are 
- iBazel successfully pick up the changes and update the bundle on disk, which in turns when you refresh your browser loads the new bundle correctly. 

@valerybugakov now the problem is that any change is actually triggering a full bundle rebuild. We think that's perfectly fine (assuming we fix the images thing) when done in CI but that's not very practical in local env, where you want things to be quick. 

You can't wait 50 seconds every time you change a single js file. For now, it sounds that having the webpack server running is the way to go, at least in local env. 

How to test this branch: `go run ./dev/sg start oss-bazel`. This branch assumes everything has been previously build before at least once, so run `bazel build //...` first if you see errors.

### CODE REVIEW

File: BUILD.bazel [hunk 1]
```diff
@@ -15,6 +15,7 @@ load("@npm//:defs.bzl", "npm_link_all_packages")
 
 # Enable only the Aspect javascript plugin for gazelle
 # gazelle:lang js
+# gazelle:lang go
```
Comment (jhchabran) [inline BUILD.bazel:18]: This got us to pull our hairs for a few minutes :D

File: sg.config.yaml [hunk 2]
```diff
@@ -871,8 +872,80 @@ commands:
       - internal
       - schema
 
+bazelCommands:
+  oss-web:
```
Comment (jhchabran) [inline sg.config.yaml:876]: This builds the bundle.

File: ui/assets/assets_dev.go [hunk 3]
```diff
@@ -5,4 +5,4 @@ package assets
 
 import "net/http"
 
-var Assets = http.Dir("./ui/assets")
+var Assets = http.Dir("./bazel-bin/client/web/bundle")
```
Comment (jhchabran) [inline ui/assets/assets_dev.go:8]: This sounds a bit rough, but it does the job for now.

File: sg.config.yaml [hunk 4]
```diff
@@ -871,8 +872,80 @@ commands:
...
+      - oss-frontend
+      - oss-worker
+      - oss-repo-updater
+      - oss-symbols
+      - oss-gitserver-0
+      - oss-gitserver-1
+      - github-proxy
+      - searcher
+    commands:
+      # - oss-web
```
Comment (jhchabran) [inline sg.config.yaml:940]: This is the normal webpack bundle server, which listens on 3080.

File: BUILD.bazel (hunk 1 above)
Comment (valerybugakov) [reply BUILD.bazel:18]: Ah, I see. I'll keep an eye on that too!

Comment (valerybugakov) [reply BUILD.bazel:18]: Heads up: https://github.com/sourcegraph/sourcegraph/pull/48429/files#r1121550982

**Pull request closed without merging**